app/views/admin/loans/show.html.slim

```diff
@@ -18,6 +18,7 @@ javascript:
       el: '.loan-content',
       calendarEventsUrl: #{json @calendar_events_url},
       loanId: #{json @loan.id},
-      locale: "#{I18n.locale}"
+      locale: "#{I18n.locale}",
+      questionnaireFilter: #{json @questionnaire_filter}
```

Comment:
Because of the current tab structure, the initial questionnaire view model is initialized inside the loan tabs view model. We can clean this up, but the tab restructuring is not in yet.